### DISCUSSION TITLE
can i use co-pilot on multiple accounts?

### DISCUSSION THREAD

**Discussion by slim-python** [community/community#46949] [Copilot Conversations]
### Select Topic Area

Question

### Body

i have two github account (one is for personal one is for office, office one uses bitbucket)
if i buy the github co -pilot from my personal github account, will i be able to use in both the accounts?

right now i use bitbucket with my office account, so how will i be able to use it there? i heard you have to sign in with your github to be able to use it

**Comment by jge162** [comment]
I believe that is against TOS. But check documentation to verify. In regards to using copilot on shared accounts with one subscription.

**Reply by jge162** [reply]
See [community post](https://github.com/community/community/discussions/12157) here where dev says it is against TOS to share.

**Comment by iamcjwilliams** [comment]
@slim-python Visit [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue

**Comment by priya-jain-dingg** [comment]
I have same issue. Any update on this.

**Comment by alejandrod** [comment]
This is too bad!! Why would github assume that there is a single handle?
If you are a contractor and work for multiple companies, you might have multiple handles as each company might require it's own handle.
Why would you pay for each one?

**Reply by VTJ-Dumay-Pothuon** [reply]
Easy money. For them.

**Reply by jasonwilliamspaylinksolutions** [reply]
I agree I have the same issue and I presume many engineers will. It's a show stopper for me until they address that issue.

**Reply by vnnegt** [reply]
agreed - i wouldn't have purchased if i knew it couldnt be shared across multiple accounts

there can't be that many professional developers that only have 1 github account

**Comment by flicwin** [comment]
GitHub, this is such a poor outcome for what you tout as being the next big thing in coding assistance. As a freelancer, this product is pointless. Very disappointed. I'm not made of the money you seem to think we can all cough up for multiple subscriptions - and multiple subscriptions are counter-productive if half the features are isolated to each instance.

**Comment by drackp2m** [comment]
Anyone can invite any user to their private projects to collaborate. Even at work, I use my personal GitHub account to log in to VSCode, where I have several profiles, depending on the environment (work or personal), I have some extensions or others installed, different themes, etc. And Copilot works perfectly, both at home and at work.

Not to mention the recent release (finally) of Copilot on the web. The truth is, I would also like to be able to use it on several accounts, but mainly, it would be to share it with my wife, who is starting to study, and could really use the help. And surprise, this case would be a violation of the terms. If they left that much freedom, things would quickly get out of control for them.

As I say, I would also like it to work as you mentioned, but I understand the reason why it doesn't, and I resign myself. But I don't regret paying for my annual license.
